OpenUtau/Strings/Strings.axaml

```diff
@@ -225,7 +225,8 @@ Warning: this option removes custom presets.</system:String>
   <system:String x:Key="pianoroll.tool.drawpitch">Draw Pitch Tool (4)
     Left click to draw
     Right click to reset
-Hold Ctrl to smoothen</system:String>
+    Hold Ctrl to select
+    Hold Alt to smoothen</system:String>
```

Comment:
When updating a string, delete it from all languages so that translators will know it needs to be translated again. Otherwise, the wrong texts will not be updated.

Reply:
Fixed translations.